### PR TITLE
Added week 1 and 2 files

### LINKED ISSUE #46: [Feature Request] Implement Weeks 1 and 2 of GameDev BitCamp
**Is your proposal related to a problem?**

Updating the GameDev curriculum

**Describe the solution you'd like**

Upload Weeks 1 and 2 of the GameDev Curriculum

**Additional context**

Weeks 1 and 2 of 7, as part of BitCamp GameDev.

### PR DESCRIPTION
**Pull request by MurraySmith27**
Added the files for week 1 and week 2 live streams and homework.

### CODE REVIEW

**Review by beaubhp — CHANGES_REQUESTED** [review]
For `week1/livestream/script.md`, I would format it how you did it for Week 2 in more of a bulleted list format. It is just easier to read and looks much better!

**Review by beaubhp — COMMENTED** [review]
Other than a few styling changes (with markdown), the content itself looks great.

File: GameDev/week1/livestream/script.md [hunk 1]
```diff
@@ -1 +1,12 @@
-Put your script here.
```
Comment (emsesc) [inline GameDev/week1/livestream/script.md:1]: Agree with @Bahburs -- some styling with markdown will improve readability

File: GameDev/week2/homework/issues/Week 2 Homework Instructions.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Bitcamp Game Development - Week 2 Homework
+
+For this week’s homework, we’ll be adding assets to our game and writing basic scripts in C#.
+
+Your task is to create a road race! To do this, you’ll have to start from the project we installed in the first week (the one with the cars and the road). Try to do these steps on your own first, but if you need a little guidance, check the unity C# documentation or the recording from this week’s live session.
+
```
Comment (emsesc) [inline GameDev/week2/homework/issues/Week 2 Homework Instructions.md:6]: same issue with styling (this is better than the live stream script formatting though)
example: https://github.com/TheDirector23/azure-functions-course/blob/main/responses/2.0-postman.md
markdown cheatsheet: https://www.markdownguide.org/cheat-sheet/

**Review by emsesc — CHANGES_REQUESTED** [review]
Overall, great content! I would just change the formatting with markdown to make it more readable. I would also recommend transferring the content in the solutions pdfs into a markdown file. That way, in case changes need to be made, they can be committed in Github.

**Comment by MurraySmith27** [comment]
Okay I made some formatting changes. Thanks for the feedback!

**Comment by lazyplatypus** [comment]
@emsesc make sure to attach the issue here :)

**Comment by emsesc** [comment]
@MurraySmith27 Looks much better!

**Pull request merged**